index.js

```diff
@@ -76,8 +76,17 @@ module.exports = function base (ALPHABET) {
     return bytes.reverse()
   }
 
+  function isValid (string) {
+    for (var i = 0; i < string.length; i++) {
+      if (ALPHABET_MAP[string[i]] === undefined) return false
+    }
+
+    return true
+  }
```

Comment:
Technically it should be fine, validating that each element is part of the alphabet.

However given issues like https://github.com/cryptocoinjs/base-x/issues/18 it might be more complicated?